pdf-generator/build.gradle

```diff
@@ -22,9 +22,12 @@ dependencies {
   }
   compile group: 'com.github.librepdf', name: 'openpdf', version: openPdf
 
+  compile group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '2.+'
```

Comment:
does this not come as a dependency of java-logging-appinsights?

Reply:
removed ... I wasn't aware of this based module.